.github/workflows/envoy-sync-scheduled.yaml

```diff
@@ -0,0 +1,53 @@
+name: Sync from Upstream (Scheduled)
+
+permissions:
+  contents: read
+
+on:
+  schedule:
+    - cron: "0 1 * * *"
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+
+jobs:
+  sync:
+    if: github.repository == 'envoyproxy/envoy-openssl'
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        branch_name:
+          - release/v1.32
+          - release/v1.28
```

Comment:
nit: do we need 1.28?

Reply:
We still support it so I figured it was best to keep it up to date? Do you think this might cause a problem?

Reply:
ah ok, I didn't know 1.28 is supported.